frontend/src/metabase/admin/routes.jsx

```diff
@@ -182,6 +182,13 @@ const getRoutes = (store, CanAccessSettings, IsAdmin) => (
           ))}
         </Route>
       </Route>
+      {PLUGIN_METABOT.AdminRoute}
```

Comment:
I'm really proud of this method of injecting plugin routes. It's much easier to understand than manipulating an array in some other piece of code